torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -745,6 +751,8 @@ def rewrite_node(node):
                     or full_pad[-2][1] % ttnn.TILE_SIZE != 0
                 ):
                     input = g.call_function(ttnn.to_layout, args=(input, TtnnRowMajorLayout()))
+                if output_shape[-1] % 2 != 0:
```

Comment:
This must be very easy to fix now that we allow odd number values on the innermost  dimension in bfloat16